kolibri/core/content/views.py

```diff
@@ -111,7 +112,12 @@ def replace_script(parent, script):
 
 def parse_html(content):
     try:
-        document = html5lib.parse(content, namespaceHTMLElements=False)
+        # We need the reference to the HTMLParser later to get the document encoding,
+        # so we can't use the html5lib.parse convenience function here.
+        tb = html5lib.treebuilders.getTreeBuilder("etree")
+        p = html5lib.HTMLParser(tb, namespaceHTMLElements=False)
+        document = p.parse(content)
```

Comment:
could we give these more descriptive names? e.g.


```suggestion
        # We need the reference to the HTMLParser later to get the document encoding,
        # so we can't use the html5lib.parse convenience function here.
        tree_builder = html5lib.treebuilders.getTreeBuilder("etree")
        parser = html5lib.HTMLParser(tree_builder, namespaceHTMLElements=False)
        document = p.parse(content)
```

Reply:
Yeah, can do!